### PR TITLE
BC-8342 - Change naming from participants to members

### PR DESCRIPTION
**Pull request by muratmerdoglu-dp**
# Short Description

## Links to Ticket and related Pull-Requests
- https://ticketsystem.dbildungscloud.de/browse/BC-8342

## Checklist before merging

- [ ] QA: In addition to review, the code has been manually tested (if manual testing is possible)
- [x] PO: Any deviation from requirements was agreed with Product-Owner / ticket author / support-team
- [x] DEV: Every new component is implemented having accessibility in mind (e.g. aria-label, role property)

> Notice: Please keep this Pull-Request as a Draft (or add WIP label), until it is ready to be reviewed

### CODE REVIEW

File: src/modules/feature/room/RoomMembers/MembersTable.vue [hunk 1]
```diff
@@ -19,28 +19,26 @@
...
-				t('pages.rooms.participants.participantTable.itemsPerPage')
-			"
 			:no-data-text="t('common.nodata')"
 			:mobile="null"
 			mobile-breakpoint="sm"
 			@update:current-items="onUpdateFilter"
 		>
 			<template #[`item.actions`]="{ item }">
-				<v-btn
+				<v-icon
```
Comment (NFriedo) [inline src/modules/feature/room/RoomMembers/MembersTable.vue:36]: This should be a button (was introduced in https://github.com/hpi-schul-cloud/nuxt-client/pull/3439 and discussed with UX :) ) 
So `class=cursor-pointer` can also be removed when using an icon button.

Probably was messed up a little bit when merging main into this branch.

File: src/modules/feature/room/RoomMembers/MembersTable.vue [hunk 2]
```diff
@@ -100,7 +97,6 @@ const tableHeader = [
 		title: t("common.labels.role"),
 		key: "displayRoleName",
 	},
-	// TODO: Decide if we want to show classes in the table or not
 	// {
```
Comment (NFriedo) [inline src/modules/feature/room/RoomMembers/MembersTable.vue:100]: I think when removing the comment `TODO: Decide if we want to show classes in the table or not` we could also remove the other comments (lines 100-102). We can add them again when really introducing classes to the table :)

File: src/modules/page/room/RoomMembers.page.unit.ts
Comment (NFriedo) [inline src/modules/page/room/RoomMembers.page.unit.ts:1]: I think we should fix the vue warnings in this test ` [Vue warn]: Failed to resolve directive: dompurify-html `, we could add `vueDompurifyHTMLPlugin` to the plugins in the setup.

File: src/modules/page/room/RoomMembers.page.unit.ts [hunk 3]
```diff
@@ -259,47 +243,47 @@ describe("RoomParticipantsPage", () => {
...
 			});
-			expect(participantsTable.exists()).toBe(true);
+			expect(membersTable.exists()).toBe(true);
 		});
 	});
 
-	describe("AddParticipant Dialog", () => {
-		it("should render AddParticipants", async () => {
+	describe("AddMembers Dialog", () => {
+		it("should render AddMembers", async () => {
```
Comment (NFriedo) [inline src/modules/page/room/RoomMembers.page.unit.ts:257]: I think "should open dialog" or "should open AddMembers dialog" (like line 247 for close dialog)  would be better.
I think just "should render AddMembers" is not so clear :)

**Review by NFriedo — CHANGES_REQUESTED** [review]

File: src/locales/de.ts [hunk 4]
```diff
@@ -1673,25 +1673,23 @@ export default {
...
-		"Einträge pro Seite",
-	"pages.rooms.participants.roles.editor": "Raumeditor",
-	"pages.rooms.participants.roles.viewer": "Raumbetrachter",
+	"pages.rooms.member.label": "Teilnehmer:in",
+	"pages.rooms.members.label": "Teilnehmende",
+	"pages.rooms.members.add": "Teilnehmende hinzufügen",
+	"pages.rooms.members.manage": "Teilnehmende verwalten",
+	"pages.rooms.members.remove": "Teilnehmende entfernen",
+	"pages.rooms.members.roles.editor": "Raumeditor",
+	"pages.rooms.members.roles.viewer": "Raumbetrachter",
```
[the remark this answers is not in the document]
Comment (muratmerdoglu-dp) [reply src/locales/de.ts:1692]: They can be used in our next sprints, so it's better to keep them.

**Review by NFriedo — APPROVED** [review]

**Pull request closed without merging**